source/frontend/src/features/mapSideBar/research/ResearchContainer.tsx

```diff
@@ -18,26 +14,20 @@ import { ApiGen_Concepts_File } from '@/models/api/generated/ApiGen_Concepts_Fil
 import { ApiGen_Concepts_ResearchFile } from '@/models/api/generated/ApiGen_Concepts_ResearchFile';
 import { UserOverrideCode } from '@/models/api/UserOverrideCode';
 import { exists, stripTrailingSlash } from '@/utils';
-import { getFilePropertyName } from '@/utils/mapPropertyUtils';
 
 import { SideBarContext } from '../context/sidebarContext';
-import SidebarFooter from '../shared/SidebarFooter';
-import { UpdateProperties } from '../shared/update/properties/UpdateProperties';
-import ResearchHeader from './common/ResearchHeader';
-import ResearchMenu from './common/ResearchMenu';
 import { useGetResearch } from './hooks/useGetResearch';
 import { useUpdateResearchProperties } from './hooks/useUpdateResearchProperties';
-import ResearchView from './ResearchView';
+import { IResearchViewProps } from './ResearchView';
 
 export interface IResearchContainerProps {
   researchFileId: number;
   onClose: () => void;
+  View: React.FunctionComponent<React.PropsWithChildren<IResearchViewProps>>;
```

Comment:
I refactored the `ResearchContainer` to have the view injected via dependency injection (like other file containers in the code base). This makes testing easier and improves code maintainability.